src/components/MySpace/MySpace.stories.tsx

```diff
@@ -19,26 +19,26 @@ export default {
 const exampleMySpaceData = [
   {
     __typename: 'Collection',
-    _id: new ObjectId(),
+    _id: new ObjectId().toHexString(),
```

Comment:
Custom Collection was broken in Storybook. Needed to fix the _ids and now it should display as expected